### PR TITLE
Return behavior objects from gamelab_behavior_get

### PR DESCRIPTION
**Pull request by balderdash**
I already started storing the behaviors on a sprite as objects with `func` and `extraArgs` properties, and when bare functions are passed into `addBehavior` or `removeBehavior` they get normalized into that format.

This PR updates the `gamelab_behavior_get` block to output a behavior object instead of a bare function. Once we migrate the rest of the old behaviors to be blockly-defined, we can drop support for bare functions across the behavior handling code.

### CODE REVIEW

File: apps/src/gamelab/GameLabJr.interpreted.js [hunk 1]
```diff
@@ -67,12 +67,16 @@ function removeBehavior(sprite, behavior) {
...
+  };
+}
+
 function normalizeBehavior(behavior) {
   if (typeof behavior === 'function')  {
-    behavior = {
-      func: behavior,
-      extraArgs: [],
-    };
+    return createBehavior(behavior, []);
```
Comment (joshlory) [inline apps/src/gamelab/GameLabJr.interpreted.js]: Do you think we'll ever want instance methods on these behavior hashes?  If so, maybe prefer:

```js
function Behavior(func, extraArgs) {
  this.func = func;
  this.extraArgs = extraArgs;
}

// Usage
new Behavior(behavior, []);
```

File: apps/src/gamelab/blocks.js [hunk 2]
```diff
@@ -257,10 +257,13 @@ export default {
     };
 
     generator.gamelab_behavior_get = function () {
-      return [
-        Blockly.JavaScript.variableDB_.getName(
+      const name = Blockly.JavaScript.variableDB_.getName(
             this.getTitleValue('VAR'),
-            Blockly.Procedures.NAME_TYPE),
+            Blockly.Procedures.NAME_TYPE);
+      // TODO: add support for passing extra params into this block
```
Comment (joshlory) [inline apps/src/gamelab/blocks.js:263]: TODO

Comment (balderdash) [reply apps/src/gamelab/blocks.js:263]: Have you had started working on this already? It's closely related to your behavior editor changes.

File: apps/src/gamelab/GameLabJr.interpreted.js (hunk 1 above)
Comment (balderdash) [reply apps/src/gamelab/GameLabJr.interpreted.js]: I'm not sure if we'll ever need instance methods, but I think I like that better anyway.

File: apps/src/gamelab/blocks.js (hunk 2 above)
Comment (joshlory) [reply apps/src/gamelab/blocks.js:263]: Are these the extra params beyond the `this sprite` param?  I only mutate the registered call blocks, via [`updateCallerParams_`](https://github.com/code-dot-org/blockly/blob/868084e83cc33d24b992039f40a6b04728aeb7d4/blocks/procedures.js#L180-L187).

Does this block implement [`setProcedureParameters`](https://github.com/code-dot-org/blockly/blob/04705fd7494610e1f6a2b682b3f33ebbc3fb3161/core/utils/procedures.js#L294-L299)?

Comment (balderdash) [reply apps/src/gamelab/blocks.js:263]: Correct, it's the extra params beyond `this sprite`. It does not implement `setProcedureParameters` yet.

**Pull request merged**